data/json/items/armor/storage.json

```
@@ -808,6 +808,36 @@
     "material_thickness": 2,
     "flags": [ "VARSIZE", "WATER_FRIENDLY", "BELTED" ]
   },
+  {
+    "id": "armrig",
+    "type": "ARMOR",
+    "name": { "str": "armband pouch", "str_pl": "armband pouches" },
+    "description": "A small pouch that can be worn on the upper arm using buckled straps.  This is a favoured item among sports & camping enthusiasts.",
+    "weight": "205 g",
+    "volume": "550 ml",
+    "price": 3000,
+    "price_postapoc": 250,
+    "material": [ "cotton" ],
```

```suggestion
    "material": [ "cotton", "plastic" ],
```
Or "neoprene", perhaps.

---

sure why not